src/styles/index.scss

```diff
@@ -2,2 +1,231 @@
+.page {
   margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+:root {
+  --main-text-color: #00acdc;
+}
+
+.header {
+  margin: 0;
+  text-transform: uppercase;
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` property is redundant here as it should be applied to all elements. This is a Code Style issue.